en/docs/setup/running-the-product.md

```diff
@@ -62,13 +50,11 @@ To start and stop the server in the background mode of Linux, run
     ```
     
 1.  Open a command prompt:
-    -   On Windows, choose **Start -\> Run**, type
+    -   Windows: **Start -\> Run**, type
         `            cmd           ` at the prompt, and press Enter.
-    -   On Linux/Mac OS, establish a SSH connection to the server or log
+    -   Linux/Mac OS: establish a SSH connection to the server or log
         in to the text Linux console.
-2.  Execute one of the following commands, where
-    `          <IS_HOME>         ` is the directory where you installed
-    the product distribution:
+2.  Execute one of the following commands, to start the server. 
```

Comment:
Isn't it better to clarify that what we refer to as <IS_HOME> is the directory where the product is installed? Since this is a first timer's document, it would be better to mention that.

Reply:
This has mentioned in a previous doc. Therefore, I think we do not have to repeat it again.